frb/builds/build_hosts.py

```diff
@@ -1760,41 +2085,836 @@ def build_host_200430(build_ppxf=False, build_photom=False, build_cigale=False,
+    # Load
+    photom = Table.read(photom_file, format=frbphotom.table_format)
+    # Dust correct
+    EBV = nebular.get_ebv(gal_coord)['meanValue']  # 0.061
+    frbphotom.correct_photom_table(photom, EBV, 'HG200906')
+    # Parse
+    host200906.parse_photom(photom, EBV=EBV)
+    #print(host200906.photom.keys())
+    
+    # CIGALE
+    cigale_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 'HG200906_CIGALE.fits')
+    sfh_file = cigale_file.replace('CIGALE', 'CIGALE_SFH')
+    if build_cigale:
+        # Prep
+        #cut_photom = Table()
+        # Let's stick to DES only
+        #for key in host200906.photom.keys():
+         #   if 'DES' not in key:
+         #       continue
+         #   cut_photom[key] = [host200906.photom[key]]
+        # Run
+        #print(cut_photom)
+        print("Running cigale")
+        #cut_photom = None
+        # Run
+        print(host200906.z)
+        cigale.host_run(host200906,cigale_file=cigale_file) #, cut_photom=cut_photom, cigale_file=cigale_file)
+    # Parse
+    #warnings.warn("TURN 200906 CIGALE back on")
+    host200906.parse_cigale(cigale_file, sfh_file=sfh_file)
+    
+
+    # PPXF
+    ppxf_results_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 
+                                     'HG200906_DEIMOS_ppxf.ecsv')
+    spec_fit_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 
+                                 'HG200906_DEIMOS_ppxf.fits')
+    if build_ppxf:
+        meta, spectrum = host200906.get_metaspec(instr='DEIMOS', 
+                                                 specdb_file=os.path.join(
+                                                 os.getenv('SPECDB'), 
+                                                 'specDB_CRAFT.hdf5'))
+        # Correct for Galactic extinction
+        AV = EBV * 3.1  # RV
+        Al = extinction.ccm89(spectrum.wavelength.value, AV, 3.1)
+        # New spec
+        new_flux = spectrum.flux * 10**(Al/2.5)
+        new_sig = spectrum.sig * 10**(Al/2.5)
+        new_spec = XSpectrum1D.from_tuple((spectrum.wavelength, new_flux, new_sig))
+        #
+        R = 1607.
+
+        ppxf.run(new_spec, R, host200906.z, results_file=ppxf_results_file, 
+                 spec_fit=spec_fit_file,
+                 atmos=[[3000., 5000.], [7580, 7750.]],
+                 gaps=[[6675., 6725.]], chk=True)
+    host200906.parse_ppxf(ppxf_results_file)
+    
+
+    # Derived quantities
+
+    # AV
+    host200906.calc_nebular_AV('Ha/Hb')
+
+    # SFR
+    host200906.calc_nebular_SFR('Ha')
+
+    # Galfit
+    #host191001.parse_galfit(os.path.join(db_path, 'CRAFT', 'Heintz2020',
+    #                                     'HG191001_VLT_i_galfit.fits'))
+
+    # Vet all
+    assert host200906.vet_all()
+
+
+    # Write -- BUT DO NOT ADD TO REPO (YET)
+    path = resource_filename('frb', 'data/Galaxies/{}'.format(frbname))
+    host200906.write_to_json(path=path)
+
+
+def build_host_201124(build_ppxf=False, 
```

Comment:
as with the FRBs.  It looks like we have 2 of these.
Remove the one we don't want.

Reply:
20201124a removed.